draft-ietf-jsonpath-base.md

```diff
@@ -1203,11 +1203,15 @@ A relative path, beginning with `@`, refers to the current array element or memb
 filter selector iterates over the array or object.
 
 A singular path by itself in a Boolean context is an existence test which yields true if the path selects a node and yields false if the path does not select a node.
-To be more specific about the actual value of a node selected by a path, explicit comparisons are necessary. This existence test — as an exception to the general rule — also works with nodes with structured values.
+This existence test — as an exception to the general rule — also works with nodes with structured values.
+To make use of the JSON value of a node selected by a path, explicit comparisons are necessary.
```

Comment:
I find this really hard to understand. Might an example help?

Reply:
Good idea. I added an example in af06ecf5e3c32f28156d2a39fd9d1ecfd354e5fa and also talk about "the value of a node" rather than suddenly talking about "the JSON value of a node" which seems a little distracting. See what you think.

Reply:
I believe this is now addressed; dismissing this so we can progress.